src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -38,7 +42,12 @@ public boolean publish(String message) {
     public boolean publish(String message, String color) {
         boolean result = true;
         for (String roomId : roomIds) {
+
             String url = "https://" + teamDomain + "." + host + "/services/hooks/jenkins-ci?token=" + token;
+            if (StringUtils.isEmpty(baseUrl)) {
```

Comment:
What's the point of testing `StringUtils.isEmpty(baseUrl)` if it evaluates to true then use `baseUrl`?